packages/flutter_rearch/lib/src/side_effects.dart

```diff
@@ -6,6 +6,7 @@ import 'package:rearch/rearch.dart';
 
 part 'side_effects/animation.dart';
 part 'side_effects/keep_alive.dart';
+part 'side_effects/page_controller.dart';
```

Comment:
A little bit of a nit, but would you mind keeping this side effect in the same file? I was using the new part files for the side effects that are more complex/need some boilerplate but this one is fairly trivial